src/modules/new-request-form/translations/en-us.yml

```
@@ -0,0 +1,134 @@
+      title: "Title of the notification shown when the upload of an attachment failed"
+      screenshot: "https://drive.google.com/file/d/19pYyTsdX_PzxZpMmN3-1sQn7gG_3bxl6/view?usp=drive_link"
+      value: "Upload error"
+  - translation:
+      key: "new-request-form.attachments.upload-error-description"
+      title: "Description of the notification shown when the upload of an attachment failed"
+      screenshot: "https://drive.google.com/file/d/19pYyTsdX_PzxZpMmN3-1sQn7gG_3bxl6/view?usp=drive_link"
+      value: "There was an error uploading {{fileName}}. Please try again or upload another file."
+  - translation:
+      key: "new-request-form.attachments.drop-files-label"
+      title: "Label shown when a file is dragged over the attachment field"
+      screenshot: "https://drive.google.com/file/d/1T_2Eahrfs8iITZWvgAR6wxrxEdsy2D2G/view?usp=drive_link"
+      value: "Drop files here"
+  - translation:
+      key: "new-request-form.attachments.choose-file-label"
+      title: "Label for the attachments field"
+      screenshot: "https://drive.google.com/file/d/1T_2Eahrfs8iITZWvgAR6wxrxEdsy2D2G/view?usp=drive_link"
+      value: "Choose a file or drag and drop here"
+  - translation:
+      key: "new-request-form.attachments.stop-upload"
+      title: "Tooltip for the button for stopping the upload of an attachment"
+      screenshot: "https://drive.google.com/file/d/1Y6yeqp3fbfJbuq7tT8VutsLa80GZju-V/view?usp=drive_link"
+      value: "Stop upload"
+  - translation:
+      key: "new-request-form.attachments.remove-file"
+      title: "Tooltip for the button for removing an attachment"
+      screenshot: "https://drive.google.com/file/d/1JFVE2twzUkg-U6gGAs2EHqD-3jPvtJDA/view?usp=drive_link"
+      value: "Remove file"
+  - translation:
+      key: "new-request-form.attachments.uploading"
+      title: "[A11Y] Hidden label for screen readers for the upload attachment progress bar"
+      screenshot: "https://drive.google.com/file/d/1Y6yeqp3fbfJbuq7tT8VutsLa80GZju-V/view?usp=drive_link"
+      value: "Uploading {{fileName}}"
+  - translation:
+      key: "new-request-form.submit"
+      title: "Submit button for the New Request Form"
+      screenshot: "https://drive.google.com/file/d/1VuVYpxGA-uAGbvscl2WXTJ24yK5QydhZ/view?usp=drive_link"
+      value: "Submit"
+  - translation:
+      key: "new-request-form.cc-field.email-removed"
+      title: "[A11Y] Announcement read by screen readers when an e-mail is removed from the CC field."
+      screenshot: "https://drive.google.com/file/d/19iWawEvm6PJw1D2M7I1b32ntMDb7GqNw/view?usp=drive_link"
+      value: "{{email}} has been removed"
+  - translation:
+      key: "new-request-form.cc-field.emails-added"
+      title: "[A11Y] Announcement read by screen readers when an e-mail is added to the CC field."
+      screenshot: "https://drive.google.com/file/d/19iWawEvm6PJw1D2M7I1b32ntMDb7GqNw/view?usp=drive_link"
+      value: "{{emails}} have been added"
```

```suggestion
      value: "{{email}} has been added"
```
Looking at the title field, think you meant to have the singular version here, or vice-versa? The key name is for the plural though 😄